lib/crowbar/client/command/proposal.rb

```diff
@@ -22,10 +24,16 @@ module Proposal
 
         included do
           desc "Proposal specific commands"
-          command :proposal do |parent|
+          command "proposal" do |parent|
             parent.desc "Show a list of available proposals"
             parent.arg :barclamp
-            parent.command :list do |c|
+            parent.command "list" do |c|
+              c.desc "Format of the resulting output"
+              c.flag [:format], type: String, default_value: :table
+
+              c.desc "Filter output by criteria"
```

Comment:
There's probably a standard way of saying it, but something like "Filter out output not matching FILTER" (with FILTER being shown as the option passed to the flag -- not sure if that works with the ruby gem you're using for this, see "osc sr --help" to get an idea if what I mean)